### PR TITLE
Add tag management page

### LINKED ISSUE #3058: [feature] Rename a label
At the moment, a label can be created or deleted. A rename action would be a nice addition.

### PR DESCRIPTION
**Pull request by aledeg**
Closes #3058

Changes proposed in this pull request:

- The new page allows to create, delete and rename tags.

How to test the feature manually:

1. access the tag management page
2. create a tag
3. rename a tag
1. delete a tag

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

**Comment by Alkarex** [comment]
Sorry for the delay @aledeg . Looks great 👍

**Comment by Alkarex** [comment]
For another PR, maybe we should update the corresponding menu, but I am not sure what it should be (link to configuration page maybe?):
image

Ah, and maybe we could move the new menu higher, just underneath the category management:
image

**Comment by Frenzie** [comment]
I overlooked it, but tags management is the same kind of Frenglish as subscriptions management. I'll create a quick PR. ;-)

**Comment by aledeg** [comment]
@Frenzie I am guilty. I didn't think much and got inspired by the other label

**Comment by aledeg** [comment]
@Alkarex Don't worry. We all have a life outside FRSS :)

File: app/Controllers/tagController.php [hunk 1]
```diff
@@ -77,4 +77,42 @@ public function getTagsForEntryAction() {
...
+		$tagDAO = FreshRSS_Factory::createTagDao();
+		$newTag = $tagDAO->searchByName($name);
+		if (null === $newTag) {
+			$tagDAO->updateTag(Minz_Request::param('id_tag'), ['name' => $name]);
+		} else {
+			$tagDAO->updateEntryTag(Minz_Request::param('id_tag'), $newTag->id());
+			$tagDAO->deleteTag(Minz_Request::param('id_tag'));
+		}
+
+		Minz_Request::good('feedback.tag.renamed', ['c' => 'tag', 'a' => 'index'], true);
```
Comment (Alkarex) [inline app/Controllers/tagController.php:111]: The i18n for those strings `feedback.tag.*` do not seem to exist

**Pull request merged**